app/models/user.rb

```diff
@@ -8,7 +8,7 @@ class User < ApplicationRecord
   default_scope { order(:name) }
 
   has_many :schedules, dependent: :destroy
-  has_many :bookable_slots
+  has_many :bookable_slots, dependent: :destroy, foreign_key: 'guider_id'
```

Comment:
Sorry to be a super-pest but can you provide the foreign key value as a symbol per the line below? 🕵️‍♀️